src/chain/state.rs

```diff
@@ -74,8 +74,9 @@ impl State {
         if new_state.height < self.consensus_state.height {
             fail!(
                 StateErrorKind::HeightRegression,
-                "last height:{} new height:{}",
+                "last height:{} in state file {:?}, new height:{}",
```

Comment:
is this needed?

Reply:
I think it's better, if we don't set the state file path in the config file, kms will give us a default one. When I reset the chain and start a new one with the same chain-id, the chain height starts with 1, and the height in state file is the old heigh `650`,  it will go into this logic and gives me a `double sign error`. It cost me a lot of time to find out where is the `650` come from, and where is the state file, so I add the the state file in the log.